.circleci/main.yml

```diff
@@ -929,12 +929,16 @@ jobs:
             npm run eslint:check
             python -m pip install -e '.[dev]'
 
+            touch /tmp/mount-server.log # mount-server will truncate on startup
             tail -F /tmp/mount-server.log &
+            export MOUNT_SERVER_LOG_FILE=/tmp/mount-server.log \
```

Comment:
Nit: this test tests our Python handlers with mocked responses so it never reaches the point where the mount server is initiated. We could leave these 3 lines as they are, remove them, or leave a comment. I'm good with whatever you decide.

Reply:
Good catch! Removed